server/views/partials/layout.njk

```diff
@@ -12,24 +12,26 @@
   {%endfor%}
 {% endblock %}
 
-{% block pageTitle %}{{pageTitle | default(applicationName)}}{% endblock %}
+{% block pageTitle %}
+  {% if errorSummaryList | length %}Error: {% endif %}{{pageTitle | default(applicationName)}}
+{% endblock %}
 
 {% block header %}
   {{ feComponents.header | safe }}
 {% endblock %}
 
-{% block beforeContent %}
-  {% if errorSummaryList.length > 0 %}
+{% block content %}
+  {% if errorSummaryList | length %}
```

Comment:
use nunjucks `length` function instead

Reply:
also error summary needs to be at the top of `main` section rather than above - see https://design-system.service.gov.uk/components/error-summary/#where-to-put-the-error-summary